gr-qtgui/docs/qtgui.dox

```diff
@@ -129,7 +129,7 @@ The QT GUI blocks require the following dependencies.
 \li QtCore (version >= 4.4)
 \li QtGui (version >= 4.4)
 \li QtOpenGL (version >= 4.4)
-\li PyQt4 for Qt4 (version >= 4.4)
+\li PyQt5 for Qt5 (version >= 5.0)
```

Comment:
What's the lowest version we are able to test with?

Reply:
I don't know -- and the current code has no Qt version requirements whatsoever. Hence, I went down to the bottom.